.github/workflows/update-frozen.yml

```diff
@@ -1,7 +1,9 @@
 name: Update Frozen Requirements
 on:
-   schedule:  # weekly
-     - cron: '0 0 * * 0'
+  schedule:  # weekly
```

Comment:
Before we close this,  I think we should modify this workflow so that every weekend GitHub does a chilly build.   That's essentially what this does now...  a floating build followed by automatically saving the resulting frozen requirements.  (What's missing is the chilly part.)   So potentially this should do a chilly swap to make the frozen requirements chilly... then do a frozen build instead of floating...  then make sure that even frozen builds re-freeze (which I believe I already changed but check).   Then the remainder of the script should save the new frozen requirements on a branch (should be close,  might be some git tweaks due to chilly swap).

Reply:
sry just re-read, so you want to do another build to make sure that chilly still works

Reply:
For the current chilly,  my concept now is to (a) fuzz the frozen requirements and then (b) do a new frozen build to see how the fuzzed requirements are updated/overwritten.   (When I last messed with this though,  I was replacing the floating requirements and doing a floating build...  which then naturally produces new frozen requirements.   A thing to check for now is that there is no logic anywhere (in the common-scripts or tools) that prevents re-freezing during a frozen build...    which is now required as I understand it.    Once new frozen requirements are produced,  they are saved to a branch on github where they can be compared to the frozen requirements in main and/or tested.     If the updates turn out to be (a) useful and (b) harmless,  our ultimate goal is automatic weekly image updates on AWS for bugfix/security purposes only.